participants/Orion-DeepLearningCloud

```diff
@@ -0,0 +1,67 @@
+# Orion Deep Learning Cloud
+
+
+## (0) Project Name and Team Slack Handles
+Orion Deep Learning Cloud
+@Charles Cao - NBFS
+
+## (1) Brief Project Description
+
+Nebula AI is a container-based cloud computing solution provider headquartered in Montreal, Canada. Orion Cloud Computing has created a complete intelligent edge cloud computing ecosystem through its own intellectual property rights distributed container cloud computing, S3/Azure cloud storage system and data center management solutions. Through the multi-center computing solution, the utilization rate of resources is maximized, saving up to 70% of operating costs for users, helping enterprises to deploy globally and integrating cross-border data center resources.
+
+Currently Orion has a General Storage Service (GS2) store unstructured data such as photos, videos, log files, backups and container images compatible with Amazon S3 cloud storage service. GS2 is a hot storage with a limitation of 5TB. With this project, we want to integrate Filecoin storage as an alternative choice of S3/Azure and also working as a cold storage solution for GS2.
+
+This project is currently under Canada Next Generation Network support.
+
+
+## (2) Link to Project UI
+https://nbai.io/dashboard
+Project Video https://youtu.be/FzFNgC4sL3g
+GS2 https://youtu.be/NhwUO2xpb-A
+
+
+## (3) What does your application/UI do?
+
+When User using Orion cloud user can create a customized user key for protect his content before upload the content. The key will be used for encrypting and decrypt the data, but he can also upload the data without encryption.  He can view the data he uploaded in the UI and downloaded to local system. He can also use another service module called Notebook for Deep learning service.
+
+## (4) If your project is using a curated dataset, which one are you using?
+It is up to the user, we have a sample code of Obama lecture for voice recognition, which is a public dataset.  https://github.com/nebulaai/voice-recognition
+Some data like 
+COVID-19 Open Research Dataset	An AI challenge with AI2, CZI, MSR, Georgetown, NIH & The White House	19 GB	JSON	https://www.kaggle.com/allen-institute-for-ai/CORD-19-research-challenge
+
+
+
+Can also be used for training,
```

Comment:
Can you add any missing datasets to datasets.md for public sources?   Note that we may need to access encrypted data for the Slingshot evaluation portion.  Please provide a mechanism to do so.

Reply:
This is the voice dataset:  https://commonvoice.mozilla.org/en/datasets
I will add it in another PR request.

For the encrypted data,his is a UI we created for user access tokens, we will add a place for them enter password to do AES  encryption,those password is encrypted in our database. We can used it for evaluation purpose only.

![image](https://user-images.githubusercontent.com/8363795/94745470-c1ddcf80-0348-11eb-9011-886e484c23c9.png)

However if they pre-encrypted data by themselves ,we cannot help.